blocks/extensions/monitoring.py

```diff
@@ -139,3 +141,32 @@ def do(self, callback_name, *args):
             self.add_records(self.main_loop.log,
                              self._buffer.get_aggregated_values().items())
             self._buffer.initialize_aggregators()
+
+
+@six.add_metaclass(ABCMeta)
+class MonitoredQuantity(object):
```

Comment:
I think that a reasonable place for `MonitoredQuantity` is `blocks/bricks/monitoring/aggregation.py`.